game-cards/SGgame.js

```diff
@@ -300,6 +300,14 @@ exports.commands = {
 			if (!user.console) return;
 			return this.sendReply('Not Avaliable');
 		},
+		home: function (target, room, user) {
+			if (!user.console) return;
+			user.console.update(false, false, false);
+		},
+		reset: function (target, room, user) {
```

Comment:
This should be built into the /playalpha command via aliases. ex: `resetalpha confirmresetalpha`